compiler/types.nim

```diff
@@ -612,13 +612,13 @@ proc firstOrd*(t: PType): BiggestInt =
     else:
       assert(t.n.sons[0].kind == nkSym)
       result = t.n.sons[0].sym.position
-  of tyGenericInst, tyDistinct, tyTypeDesc, tyAlias:
+  of tyGenericInst, tyDistinct, tyTypeDesc, tyAlias, tyStatic:
```

Comment:
Not entirely sure if this change is correct, but one of the tests failed without it

Reply:
It's probably slightly wrong, but it's better than before.